model/buildlog.js

```diff
@@ -0,0 +1,107 @@
+const mongoose = require('mongoose');
+const fs = require("fs");
+
+
+Repository = require('./repository');
+
+const buildLogSchema = mongoose.Schema({
+  repository: String,
+  buildNumber: {
+    type: Number,
+    default: 0,
+  },
+  generate: {
+    data: Buffer,
+    datetime: String,
+  },
+  ghpages: {
+    data: Buffer,
+    datetime:String,
```

Comment:
use as a date object only instead of string. so you can use this object to aggregate